projects/project6/src/main/scala/net/sansa_stack/rdf/spark/io/package.scala

```diff
@@ -0,0 +1,240 @@
+package net.sansa_stack.rdf.spark
```

Comment:
The complete file should go out since you use a dependency for sansa-rdf-spark already on your pom and it should work by default.